frontend/lib/api/labels/index.ts

```diff
@@ -0,0 +1,21 @@
+import { PollingStationType } from "@kiesraad/api";
+import { t } from "@kiesraad/i18n";
+
+export const labelForPollingStationType = withEntries<{ [K in PollingStationType]: string }>({
+  FixedLocation: t("polling_station.type.FixedLocation"),
```

Comment:
I think that this object is now a bit redundant and the labels can be used directly in the `PollingStationForm`. I'd also prefer not to use translations in the API "layer" of the frontend.

```tsx
{pollingStationTypeValues.map((pollingStationType) => (
  <ChoiceList.Radio
    key={pollingStationType}
    id={`polling_station_type-${pollingStationType}`}
    name={"polling_station_type"}
    defaultValue={pollingStationType}
    defaultChecked={pollingStation?.polling_station_type === pollingStationType}
    label={t(`polling_station.type.${pollingStationType}`)}
  />
))}

```
Ideally the `pollingStationTypeValues` comes from the openapi.ts as suggested in https://github.com/kiesraad/abacus/issues/489